src/azure-cli/azure/cli/command_modules/azure-cli-netappfiles/azure/cli/command_modules/netappfiles/_help.py

```diff
@@ -0,0 +1,583 @@
+"""
+
+helps['netappfiles account ad list'] = """
+    type: command
+    short-summary: List the active directories of an account.
+    parameters:
+        - name: --account-name --name -a -n
+          type: string
+          short-summary: The name of the ANF account
+    examples:
+        - name: Add an active directory to the account
+          text: >
+            az netappfiles account ad list -g mygroup --name myname
+"""
+
+helps['netappfiles account ad remove'] = """
+    type: command
+    short-summary: Remove an active directory from the account.
+    parameters:
+        - name: --account-name --name -a -n
+          type: string
+          short-summary: The name of the ANF account
+        - name: --active-directory
+          type: string
+          short-summary: The id of the active directory
+    examples:
+        - name: Remove an active directory from the account
+          text: >
+            az netappfiles account ad remove -g mygroup --name myname --active-directory 13641da9-c0e9-4b97-84fc-4f8014a93848
+"""
+
+
+# pools
+
+helps['netappfiles pool'] = """
+    type: group
+    short-summary: Manage Azure NetApp Files (ANF) Pool Resources.
+"""
+
+helps['netappfiles pool create'] = """
+    type: command
+    short-summary: Create a new Azure NetApp Files (ANF) pool.
+    parameters:
+        - name: --account-name -a
+          type: string
+          short-summary: The name of the ANF account
+        - name: --name --pool-name -n -p
+          type: string
+          short-summary: The name of the ANF pool
+        - name: --size
+          type: integer
+          short-summary: The size for the ANF pool. Must be an integer number of tebibytes in multiples of 4
+        - name: --service-level
+          type: string
+          short-summary: The service level for the ANF pool ["Standard"|"Premium"|"Ultra"]
+        - name: --tags
+          short-summary: Space-separated tags in `key[=value]` format
+    examples:
+        - name: Create an ANF pool
+          text: >
+            az netappfiles pool create -g mygroup --account-name myaccountname --name mypoolname -l westus2 --size 8 --service-level "Premium"
+"""
+
+helps['netappfiles pool update'] = """
+    type: command
+    short-summary: Update the tags of the specified ANF pool.
+    parameters:
+        - name: --account-name -a
+          type: string
+          short-summary: The name of the ANF account
+        - name: --name --pool-name -n -p
+          type: string
+          short-summary: The name of the ANF pool
+        - name: --size
+          type: integer
+          short-summary: The size for the ANF pool. Must be an integer number of tebibytes in multiples of 4
+        - name: --service-level
+          type: string
+          short-summary: The service level for the ANF pool ["Standard"|"Premium"|"Ultra"]
```

Comment:
These should come from reflection on an enum type, not be hard coded in help text.

Reply:
This was mentioned before but these are not an enum type internally in the RP so I'm not sure why this is necessary. Should I define a type for the commands and convert? Do you have an example for reference? This would also be a bigger change requiring updates to help and tests. I'm not sure of the value.

Reply:
https://github.com/Azure/azure-cli/blob/dev/doc/authoring_command_modules/authoring_commands.md#registering-enums

If you don't have an enum in your SDK, then 1) you should, but 2) you can use the get_enum_type method and supply a list of values. This will display them in help, permit tab completion, and handle the validation.

Reply:
ok, updated.